cpmonitor/templates/index.html

```diff
@@ -50,4 +50,12 @@ <h2 class="card-title">Klimaneutrale Städte und Kommunen</h3>
   </div>
 </div>
 
+<div class="container fixed-bottom">
+  <footer class="py-2 my-2 border-top">
+    <ul class="nav">
+      <li class="nav-item"><a class="nav-link px-2 text-muted" href="/admin">Admin-Login</a></li>
```

Comment:
`{% url 'admin:index' %}` would be the clean way, here.

Reply:
You could also enter an issue to fix all URLs in the templates an merge this for now.

Reply:
Right!
I'll try your suggestion and use that if it works, otherwise I'll add it to the existing slug issue.